app/src/main/java/com/byteshaft/callnote/NoteActivity.java

```diff
@@ -85,6 +89,12 @@ public boolean onOptionsItemSelected(MenuItem item) {
                     mDbHelpers.clickUpdate(mId, mCheckedContacts, mTitle,
                             imageVariable, mHelpers.getCurrentDateandTime());
                     mHelpers.saveSpinnerState(mTitle, spinnerState);
+                    if (vibrationSwitch.isChecked()) {
+                        vibrationValue = true;
+                    } else {
+                        vibrationValue = false;
+                    }
+                    mHelpers.saveVibrationState(mTitle+"_vibration", vibrationValue);
```

Comment:
you should change this method 'saveVibrationState' to include '_vibration' inside it so that you only have to provide the title of the note as parameter.